test/gtest/uct/test_flush.cc

```diff
@@ -153,18 +148,10 @@ class uct_flush_test : public uct_test {
     }
 
     void check_skip_test_flush_remote() {
-#ifdef HAVE_MLX4_DV
-        auto md = sender().md();
-        if (std::string(md->component->name) == "ib") {
-            auto ib_md = ucs_derived_of(md, uct_ib_md_t);
-            if (ib_md->dev.flags & UCT_IB_DEVICE_FLAG_MLX5_PRM) {
-                auto mlx5_md = ucs_derived_of(ib_md, uct_ib_mlx5_md_t);
-                if (!(mlx5_md->flags & UCT_IB_MLX5_MD_FLAG_KSM)) {
-                    UCS_TEST_SKIP_R("not supported");
-                }
-            }
+        if ((GetParam()->tl_name != "dc_mlx5") &&
+            (GetParam()->tl_name != "rc_mlx5")) {
+            UCS_TEST_SKIP_R("not supported");
```

Comment:
we could create rc_mlx5/dc_mlx5 on an md without devx (only mlx5dv) that will not have KSM so no flush_mr

Reply:
also, other transports (tcp, shared mem...) should pass this test

Reply:
ok reverted to the latest suggestion